lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -1610,7 +1619,18 @@ status_code_e s1ap_mme_handle_ue_context_release_complete(
           " UE Context Release commplete: S1 context should have been cleared "
           "for ueid " MME_UE_S1AP_ID_FMT "\n",
           (uint32_t) mme_ue_s1ap_id);
-      OAILOG_FUNC_RETURN(LOG_S1AP, RETURNerror);
+      // Dont remove UE context if UE handed over to another eNB
```

Comment:
Align the comment with the if condition
```suggestion
      // Remove UE context only if UE is not handed over to another eNB
```

Reply:
Further, the logs/comments in lines 1601-1621 can be removed since the UE context is not cleared when UE context release is sent in this new logic.